R/f.R

```diff
@@ -52,7 +52,8 @@ f_rhs <- function(f) {
 #' @rdname f_rhs
 `f_rhs<-` <- function(x, value) {
   stopifnot(is_formula(x))
-  new_f(value, f_lhs(x), f_env(x))
+  f <- new_f(value, f_env(x), f_lhs(x))
+  maybe_pattern(f, x)
```

Comment:
Maybe this would be better called `copy_lhs()`? But I'm a bit confused as to what it's doing here.